lib/install.js

```diff
@@ -55,58 +58,59 @@ async function installForPlatform (version, platform, arch) {
 
   const url = getDownloadUrl(version, platform, arch);
 
-  log.info(`Installing Chromedriver version '${version}' for platform '${platform}' and architecture '${arch}'`);
+  log(`Installing Chromedriver version '${version}' for platform '${platform}' and architecture '${arch}'`);
 
   // set up a temp file to download the chromedriver zipfile to
-  let binarySpec = `chromedriver_${platform}${arch}`;
-  log.info(`Opening temp file to write ${binarySpec} to...`);
-  let tempFile = await tempDir.open({
+  const binarySpec = `chromedriver_${platform}${arch}`;
+  log(`Opening temp file to write '${binarySpec}' to...`);
+  const tempFile = await tempDir.open({
     prefix: binarySpec,
     suffix: '.zip'
   });
+  log(`Opened temp file '${tempFile.path}'`);
 
   // actually download the zipfile and write it with appropriate perms
-  log.info(`Downloading ${url}...`);
-  let body = await request.get({url, encoding: 'binary'});
-  log.info(`Writing binary content to ${tempFile.path}...`);
+  log(`Downloading ${url}...`);
+  const body = await request.get({url, encoding: 'binary'});
+  log(`Writing binary content to ${tempFile.path}...`);
   await fs.writeFile(tempFile.path, body, {encoding: 'binary'});
   await fs.chmod(tempFile.path, 0o0644);
 
   // extract downloaded zipfile to tempdir
-  let tempUnzipped = path.resolve(path.dirname(tempFile.path), binarySpec);
-  log.info(`Extracting ${tempFile.path} to ${tempUnzipped}`);
+  const tempUnzipped = path.resolve(path.dirname(tempFile.path), binarySpec);
+  log(`Extracting ${tempFile.path} to ${tempUnzipped}`);
   await mkdirp(tempUnzipped);
   await zip.extractAllTo(tempFile.path, tempUnzipped);
-  let extractedBin = path.resolve(tempUnzipped, "chromedriver");
+  let extractedBin = path.resolve(tempUnzipped, 'chromedriver');
   if (platform === "win") {
     extractedBin += ".exe";
   }
 
   // make build dirs that will hold the chromedriver binary
-  log.info(`Creating ${path.resolve(CD_BASE_DIR, platform)}...`);
+  log(`Creating ${path.resolve(CD_BASE_DIR, platform)}...`);
   await mkdirp(path.resolve(CD_BASE_DIR, platform));
 
   // copy the extracted binary to the correct build dir
-  let newBin = await getChromedriverBinaryPath(platform, arch);
-  log.info(`Copying unzipped binary, reading from ${extractedBin}...`);
-  let binContents = await fs.readFile(extractedBin, {encoding: 'binary'});
-  log.info(`Writing to ${newBin}...`);
+  const newBin = await getChromedriverBinaryPath(platform, arch);
+  log(`Copying unzipped binary, reading from ${extractedBin}...`);
+  const binContents = await fs.readFile(extractedBin, {encoding: 'binary'});
+  log(`Writing to ${newBin}...`);
   await fs.writeFile(newBin, binContents, {encoding: 'binary', mode: 0o755});
```

Comment:
might it be that newBin location is not writeable? I'd rather catch this call then and show some helpful error message to the client